static/js/modules/election-summary.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+/* global require, module */
+
+var $ = require('jquery');
+var URI = require('URIjs');
+
+var helpers = require('./helpers');
+
+function ElectionSummary(selector, opts) {
+  this.$elm = $(selector);
+  this.opts = opts;
+
+  this.$count = this.$elm.find('.count');
+  this.$receipts = this.$elm.find('.receipts');
+  this.$disbursements = this.$elm.find('.disbursements');
+
+  this.fetch();
+  this.$elm.find('a.button--election').attr('href', this.buildUrl());
```

Comment:
Is the `a` in the selector actually needed?

I'd also suggest this should be a js-prefixed class rather then using a styling class, but will defer to you for best judgement.